src/components/Hero.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import {
+  DivStyled, H1Styled, H2Styled, Button, ButtonText, SearchText, ResultThumb,
+} from './Hero.style';
+import Table from './table.png';
+
+function Hero() {
+  return (
+    <DivStyled>
+      <H1Styled>No reactions to your reddit posts?</H1Styled>
+      <H2Styled>
+        Great timing, great results! Find the best time to post on your subreddit.
+      </H2Styled>
+      <Button to="/search?q=javascript"><ButtonText>SHOW ME THE BEST TIME</ButtonText></Button>
```

Comment:
I would put the `<ButtonText>` on a new line. But actually I don't think you need that component at all.